### DISCUSSION TITLE
What's mean of the `version` field in a Module Federation Shared Module?

### DISCUSSION THREAD

**Discussion by woochanleee** [webpack/webpack#17759] [Q&A]
This may be a very stupid question. 💩

<img width="716" alt="Screenshot 2023-10-20 at 3 31 42 PM" src="https://github.com/webpack/webpack/assets/48552260/7b654093-bbfb-4fa5-bf11-6b4e90d0a3a9">

### Q1. I'm using the Shared Module in the Module Federation Plugin, but I don't understand below description in the docs.

> It allows webpack to replace lower matching versions, but not higher. [Docs](https://webpack.js.org/plugins/module-federation-plugin/#version)

### Q2. When use `false`?

Can you explain it in more detail or share an example CSB that I can understand?
Thank you.

**Comment by alexander-akait** [comment]
Hello, you can have multiple versions of your shared modules, for example - my-admin-logic@1.0.0, my-admin-logic@2.0.0 and it allows webpack to load modules properly
